e2e/solve/fund.go

```diff
@@ -16,39 +19,92 @@ import (
 	"cosmossdk.io/math"
 )
 
-func maybeFundSolver(ctx context.Context, network netconf.ID, backends ethbackend.Backends) error {
+// erc20 tokens to fund solver with on devnet. useful for solvernet development when forking public networks.
```

Comment:
golang comments start with function or variable name, e.g.: `// testTokenAddressesToFund are the tokens ...`

Reply:
this type of refactor is subjective, please try to avoid subjective refactors.